assets/js/base/components/tabs/index.js

```diff
@@ -2,7 +2,7 @@
  * External dependencies
  */
 import { useState } from '@wordpress/element';
-import { withInstanceId } from '@wordpress/compose';
+import withComponentId from '@woocommerce/base-hocs/with-component-id';
```

Comment:
We don't have to change this back, but now that WP packages can be tree-shaken, another approach here would be do direct imports for `@wordpress/compose` similar to how we do for `@wordpress/components` (via `wordpress-components`). `withComponentId` filled a gap for us when we had no way to either directly import WP packages or tree-shake them, but it's becoming unnecessary and we _could_ retire it.

Reply:
Ill sunset it in a followup